tests/e2e/utils/reset.js

```diff
@@ -32,21 +32,37 @@ import {
 
 /**
  * Resets Site Kit using a utility plugin.
+ *
+ * @since 1.0.0
+ * @since n.e.x.t Option to reset persistent options.
+ *
+ * @param {Object} options Reset Site kit Options.
```

Comment:
`options` should be defined as an optional parameter and we can add `persistent` here as well.
```suggestion
 * @param {Object}  [options]            Reset options.
 * @param {boolean} [options.persistent] Additionally deletes persistent options.
```